plasma_framework/python_tests/tests/contracts/root_chain/test_start_in_flight_exit.py

```diff
@@ -218,70 +200,6 @@ def test_start_in_flight_exit_invalid_outputs_should_fail(testlang):
         testlang.start_in_flight_exit(spend_id)
 
 
-@pytest.mark.skip("TODO: remove test when such scenarion is covered by a test that processes exits")
-@pytest.mark.parametrize("num_inputs", [1, 2, 3, 4])
-def test_start_in_flight_exit_cancelling_standard_exits_from_inputs(testlang, num_inputs):
```

Comment:
before deleting this test, please make sure that the interaction of parallel SE and IFE is tested in processing tests. 
Now do not have such a test since the previous contract made the interaction checks at starting/piggybacking exits

Reply:
Test for that interaction is named `test_output_exited_via_se_and_ife_exits_only_once`, it was merged earlier. I uploaded a merge commit, so please check it. I think this test is enough.

Reply:
I would like to have two cases: SE from IFE input, SE from IFE output, and in each case check how it behaves when IFE exits first and then when SE exits first.

Reply:
in-flight transaction input, standard exit interaction is tricky, if SE is not challenged -> plasma broken, so that's why named tests with `does_not_break_exit_processing` suffix